manager/src/main/java/com/redhat/service/bridge/manager/ProcessorServiceImpl.java

```diff
@@ -187,6 +188,10 @@ public void deleteProcessor(String bridgeId, String processorId, String customer
         if (processor == null) {
             throw new ItemNotFoundException(String.format("Processor with id '%s' does not exist on bridge '%s' for customer '%s'", processorId, bridgeId, customerId));
         }
+        if (!isProcessorDeletable(processor)) {
+            throw new ProcessorLifecycleException("Processor could only be deleted if its in READY/FAILED state.");
+        }
+        processor.setStatus(ManagedResourceStatus.DEPROVISION);
```

Comment:
This is done here [too](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/c6eff16620815d47f74ada6483ad18d14256ca20/manager/src/main/java/com/redhat/service/bridge/manager/ProcessorServiceImpl.java#L197). Which do you want to keep?

Reply:
I had removed other duplicate statement